tests/test_tracking_cli.py

```diff
@@ -48,6 +48,7 @@ def samples_dir():
             "unittest",
         ],
         taskcluster_secret=None,
+        wandb_publication=True,
```

Comment:
is it supposed to stay on?

Reply:
No, it has been merge in #567, I'm rebasing.